### ISSUE TITLE
Handling of CM4 atmosphere / ocean model disagreements on land-sea mask via data preprocessing

### ISSUE THREAD

**Issue by jpdunc23** [ai2cm/ace#547]
https://github.com/ai2cm/full-model/pull/1510 handles this in `CoupledStepper`, which means that in a standalone pretraining to coupled fine-tuning setup the pretrained atmosphere component is trained with a slightly different ocean fraction.

**Comment by mcgibbon** [comment]
@jpdunc23 can you help me understand this issue? Is there a particular bug, and in what code? Is the issue resolved?

**Comment by jpdunc23** [comment]
We can close this, thanks.
